src/services/rocksdb/backend.rs

```diff
@@ -40,9 +40,12 @@ use crate::*;
 ///
 /// # Note
 ///
-/// The storage format for this service is not **stable** yet.
+/// OpenDAL will build rocksdb from source by default.
 ///
-/// PLEASE DON'T USE THIS SERVICE FOR PERSIST DATA.
+/// To link with existing rocksdb lib, please set one of the following:
```

Comment:
I'm not too familiar with whether this has any additional restrictions on the version of the dynamic library?

Reply:
Fixed.